operator-metadata/image.yaml

```diff
@@ -0,0 +1,47 @@
+schema_version: 1
+
+name: amq7/amq-streams-operator-metadata
+description: "AMQ Streams image containing the operator metadata"
+version: "1.3.0"
```

Comment:
Not sure if the version field will be needed or if it will need to match the operator image, I guess we will find out as we get more details from RCM